playbooks/satellite/roles/docker-container-systemd/files/Dockerfile

```diff
@@ -1,6 +1,5 @@
-FROM registry.access.redhat.com/rhel7/rhel-tools
-MAINTAINER jeder <jeder@redhat.com>
-RUN yum --disablerepo='*' --enablerepo='rhel-7-server-rpms' install -y openssh-server libselinux-ruby && yum clean all
+FROM registry.access.redhat.com/rhel7/rhel-atomic
+RUN microdnf --enablerepo='rhel-7-server-rpms' install -y openssh-server libselinux-ruby yum subscription-manager
```

Comment:
Why we want this? We definitely want yum and so in the container.

Reply:
We don't have yum in rhel-atomic-base image. What this change does is, it installs yum from the rhel-7-server-rpms using microdnf.

So, once our container image is built, we will be having yum command available to us :)